src/libcrun/cgroup-systemd.c

```diff
@@ -1627,7 +1627,7 @@ append_resources (sd_bus_message *m,
 #  undef APPEND_UINT64
 #  undef APPEND_UINT64_VALUE
 
-  if (! *devices_set && (! is_update || resources->devices))
+  if (! is_update || resources->devices)
```

Comment:
this looks like it unconditionally does the backup, even if we added bpf program above. should we save whether we did and branch on that here as well?